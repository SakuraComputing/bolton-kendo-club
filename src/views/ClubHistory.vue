<template>
    <div class="club-history">
        <h1>KNMS and Doshinkenyukai Club History</h1>
        <p>{{ history.clubHistory }}</p>
    </div>    
</template>

<script>

import axios from 'axios';

export default {
    data() {
        return {
            history: ''
        }
    },
    created: function() {
        this.fetchHistory();
    },
    methods: {
        fetchHistory() {
            axios.get('/api/club').then((response) => {
                this.history = response.data[0]
            });
        }
    }
}

</script>


<style>

.club-history {
    margin: 10% 14%;
    font-size: 120%;
}
</style>

