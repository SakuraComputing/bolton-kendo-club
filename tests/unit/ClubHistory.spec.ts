import { shallowMount } from '@vue/test-utils'
import ClubHistory from '@/views/ClubHistory.vue'

describe('ClubHistory.vue', () => {
  it('renders correctly', () => {
    const wrapper = shallowMount(ClubHistory);
    expect(wrapper).toMatchSnapshot()
  })
})
