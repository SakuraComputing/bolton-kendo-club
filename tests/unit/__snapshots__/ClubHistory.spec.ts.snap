// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ClubHistory.vue renders correctly 1`] = `
<div class="club-history">
  <h1>KNMS and Doshinkenyukai Club History</h1>
  <p>Need to add in some text around the clubs history</p>
</div>
`;
